core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -264,15 +279,20 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
                 </div>
               </div>
             </div>
-            <button class="btn open-in-editor-btn"
-                    ng-click="showExplorationEditor(exploration.id); $event.stopPropagation()">
-              Open In Editor
-            </button>
+            <a ng-href="<[createExplorationLink(exploration.id)]>">
+              <button class="btn open-in-editor-btn">
+                Open In Editor
+              </button>
+            </a>
           </md-card>
           <div ng-attr-section="'<['right-section']>">
-            <div ng-if="exploration.status === 'private'" class="exp-private-text">
-              <[publishText]>
-            </div>
+            <a ng-href="<[createExplorationLink(exploration.id)]>"
+               ng-if="exploration.status === 'private'"
+               style="text-decoration: none; display: inline-block; height: 88px">
```

Comment:
I think hard coding the height here might make the behavior to be non-consistent on smaller screens. Can you check ?